src/atomize-d.ts

```diff
@@ -0,0 +1,30 @@
+import { AutoLanguageClient } from "atom-languageclient"
+import { installServeD } from "./installation"
+
+class DLanguageClient extends AutoLanguageClient {
+  getGrammarScopes() {
+    return ["source.d", "D"]
+  }
+  getLanguageName() {
+    return "D"
+  }
+  getServerName() {
+    return "serve-d"
+  }
+
+  getConnectionType(): "stdio" {
+    return "stdio"
+  }
+
+  async startServerProcess(projectPath: string) {
+    const serveDPath = await installServeD()
+
+    const serveD = super.spawn(serveDPath, [], {
```

Comment:
there is localization support in serve-d. If atom provides the language in 2 letter format (ie. `en` or `en-GB`), then you can pass it using `["--lang", language]`

Reply:
Not so sure about that. I need to ask.